### ISSUE TITLE
Pdfpc not opening presenter window (SOLUTION)

### ISSUE THREAD

**Issue by nsauerfpe** [pdfpc/pdfpc#721]
**Expected Behaviour**
Pdfpc should open a full-screen presentation on the second monitor and a windowed presenter screen on the main monitor upon launch.

**Actual Behaviour**
A presenter view briefly flashes and then the presentation view appears fullscreen on second monitor, with nothing on main monitor.

**Steps to reproduce**

1. "system settings > desktop and dock > prefer tabs when opening documents" set to always
2. brew install pdfpc
3. launch pdfpc normally

To fix either:
1. Launch pdf PC with dimensioned window such as: pdfpc -Z 1000:1000 presentation.pdf
2. Split presentation view out of tab and drag to second monitor

OR 
1. "system settings > desktop and dock > prefer tabs when opening documents" set to never
2. launch pdfpc as normal

In short, if a mac user is having the problem where a presenter view is not showing, its probably in a tab in the maximized presentation view window.

Disable the prefer tabs option and relaunch the presentation.

**Comment by AndreasBilke** [comment]
Could you extend the `FAQ.rst` with your work around?

**Comment by AndreasBilke** [comment]
Fixed by #723
